www/_posts/2015-11-04-tools-release.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Tools Release: November 4th, 2015"
+categories: news
+tags: release tools
+---
+New versions of cordova tools are now live!
+
+* [cordova-lib@5.4.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova@5.4.0](https://www.npmjs.org/package/cordova)
+* [plugman@1.0.5](https://www.npmjs.org/package/plugman)
+* [cordova-js@4.1.2](https://www.npmjs.org/package/cordova-js)
+
+Release highlights:
+
+* Fixed issues with using **Node 4 & 5** and **npm 3**. 
+* Cordova will now auto convert old-style plugin IDs to new style plugin IDs when doing a `cordova plugin add`. This only happens if the old-style plugin ID exists in the [registry-mapper](https://github.com/stevengill/cordova-registry-mapper), it will be auto converted to the new ID and fetched from [npm](https://www.npmjs.com) instead. 
+* Cordova `CLI` will now know if a newer version is available and will prompt users to update.
+* Some core utility methods from `cordova-lib` have been moved into a new module named [`cordova-common`](https://github.com/apache/cordova-lib/tree/master/cordova-commo://github.com/apache/cordova-lib/tree/master/cordova-common). `cordova-common` is shared among `cordova-lib` and cordova platforms.
+
+To update your tools:
+
+  * If you have `cordova` installed:
+
+        npm install -g cordova@latest
+
+  * If you have `plugman` installed:
+
+        npm install -g plugman@latest
+
+<!--more-->
+# Changes include:
+
+## Platform updates
+When adding these platforms to your project, the following versions are now used by default.
+These platform versions were released recently, and the tools' defaults were updated:
```

Comment:
You are right of course. I just wanted to highlight that 3.9.2 ios is out. 

Tools defaults weren't updated but platform versions were released. No big deal here.